src/components/vars.sass

```diff
@@ -64,6 +64,11 @@ $color4-3: #edebc6
 $color4-4: #dcd898
 $color4-5: #d1cc7e
 $color4-6: #c2bc62
+$color4-7: #f3f1da
+$color4-8: #d2cd7f
+$color4-9: #ddd999
+$color4-10: #faf5e9
+$color4-11: #f1e3bd
```

Comment:
Not sure if general color tweaks are coming, but these were the colors from Michael in the UI spec and they were not currently in the project.  They're subtly different from some of the existing colors.  I don't think this PR is the place to do an overall color swap (or even investigate it) so for now we will just add these new colors.